core/src/com/unciv/logic/GameInfo.kt

```diff
@@ -129,6 +131,7 @@ class GameInfo {
      *  @throws NoSuchElementException in no-barbarians games! */
     fun getBarbarianCivilization() = getCivilization(Constants.barbarians)
     fun getDifficulty() = difficultyObject
+    fun getGameSpeed() = gameSpeedObject
```

Comment:
If there's an accessor, make the backing field private?

Reply:
Done